precompile-script.py

```diff
@@ -145,28 +146,34 @@ def convert_stream(fin, fout):
         converter.finish()
 
 
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Precompile VkRunner scripts.')
+    parser.add_argument('inputs', metavar='INPUT', type=str, nargs='+',
+                        help='an input file process')
+    parser.add_argument('-o', dest='output', metavar='OUTPUT',
+                        help='an output file or directory', required=True)
 
-parser = argparse.ArgumentParser(description='Precompile VkRunner scripts.')
-parser.add_argument('inputs', metavar='INPUT', type=str, nargs='+',
-                    help='an input file process')
-parser.add_argument('-o', dest='output', metavar='OUTPUT',
-                    help='an output file or directory', required=True)
+    parser.add_argument('-g', dest='glslang', metavar='GLSLANG',
+                        help='glslang binary path', required=False, default="./glslangValidator")
```

Comment:
Shouldn’t the default be `"glslangValidator"`? Otherwise it’s not going to look in the path for it when no `-g` option is given.

Reply:
OK